packages/clerk-js/src/ui/components/OrganizationProfile/BillingWidget.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRouter } from '../../../ui/router';
+import { useOrganizationProfileContext } from '../../contexts';
+import { AlertIcon, Flex, Link, Text } from '../../customizables';
+
+export const BillingWidget = () => {
+  const { __unstable_manageBillingUrl, __unstable_manageBillingMembersLimit } = useOrganizationProfileContext();
+  const router = useRouter();
```

Comment:
Should we turn these into props for the BillingWidget to force the outside check for them being defined?